### ISSUE TITLE
qtdeploy don't generate all binding for Android

### ISSUE THREAD

**Issue by bclermont** [therecipe/qt#131]
If I build the following project https://github.com/bclermont/therecipe-qt-issue-131/tree/master/main with `qtdeploy build android` it fail:

```
# github.com/bclermont/therecipe-qt-issue-131/sub
../sub/sub.go:11: undefined: core.QStandardPaths_StandardLocations
```

it fail this line https://github.com/bclermont/therecipe-qt-issue-131/blob/master/sub/sub.go#L11

but it do not fail at line 10, because https://github.com/bclermont/therecipe-qt-issue-131/blob/master/main/main.go#L10

I guess it fail because there is usage of `therecipe/qt` outside `github.com/bclermont/therecipe-qt-issue-131/main` or any of it's sub-directories.

**Comment by therecipe** [comment]
I will look into this.
The bug is definitely in qtminimal.
Probably because of the passive "_" import.

**Comment by bclermont** [comment]
you can remove the `_` and call that function in `sub` and it will be the same

**Comment by therecipe** [comment]
This works for me on macOS
```
go get github.com/bclermont/therecipe-qt-issue-131/main && cd $GOPATH/src/github.com/bclermont/therecipe-qt-issue-131/main && qtdeploy build android
```

I will test it on a fresh linux machine as well

**Comment by therecipe** [comment]
Works on a fresh linux box as well.
Maybe there is a problem when you use multiple GOPATH ...
I will test this.

**Comment by therecipe** [comment]
It works as well with one GOPATH containing the project and another containing the binding.
Does it fail with the same message if you run `qtminimal && go build -tags=minimal` inside main?

Maybe the `*-minimal.*` files aren't re-generated by qtminimal somehow.
Could you check the age of `qt/core/core-minimal.cpp` for example?

**Comment by bclermont** [comment]
```
export GOPATH=/path/to/first:/path/to/second
```

try first path:

```
mv /tmp/therecipe-qt-issue-131 /path/to/first/src/github.com/bclermont/
cd /path/to/first/src/github.com/bclermont/therecipe-qt-issue-131/main
qtdeploy build android
```

**works**

try second path:

```
mv /path/to/first/src/github.com/bclermont/therecipe-qt-issue-131 /path/to/second/src/github.com/bclermont/
cd /path/to/second/src/github.com/bclermont/therecipe-qt-issue-131/main
qtdeploy build android
```

**fail**

so the problem is that `qtminimal` don't support multiple dir in `GOPATH`

fixed in #132

**Comment by therecipe** [comment]
Thank you :)
I couldn't reproduce it yesterday, when I tried.
